### PR TITLE
[Bug Fix] Fix typo in RunPythonScript parameters

### PR DESCRIPTION
**Pull request by matthewfcarlson**
an x was in front of raise_exception_on_nonzero, which caused an exception

### CODE REVIEW

**Comment by spbrogan** [comment]
Can you write a unit test for this function so that these types of things are caught.

File: edk2toollib/utility_functions_test.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# @file utility_functions_test.py
+# unit test for locate_tools module.  Only runs on Windows machines.
```
Comment (spbrogan) [inline edk2toollib/utility_functions_test.py]: clean up

File: edk2toollib/utility_functions_test.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+import unittest
+import sys
+import os
+import edk2toollib.utility_functions as utilities
+
+
+class UtilityFunctionsTest(unittest.TestCase):
+
+    def test_RunPythonScript(self): # simple run yourself!
```
Comment (spbrogan) [inline edk2toollib/utility_functions_test.py]: should we add a test case that uses all named parameters

Comment (matthewfcarlson) [reply edk2toollib/utility_functions_test.py]: Added a few more test cases

File: edk2toollib/utility_functions_test.py (hunk 1 above)
Comment (matthewfcarlson) [reply edk2toollib/utility_functions_test.py]: Cleaned up

**Pull request merged**